executor/load_data.go

```diff
@@ -632,14 +632,10 @@ func (w *fieldWriter) GetField() (bool, field) {
 			}
 		} else if ch == '\\' {
 			// TODO: escape only support '\'
-			w.OutputBuf = append(w.OutputBuf, ch)
 			flag, ch = w.getChar()
 			if flag {
-				if ch == w.enclosedChar {
-					w.OutputBuf = append(w.OutputBuf, ch)
-				} else {
-					w.putback()
-				}
+				w.OutputBuf = append(w.OutputBuf, '\\')
```

Comment:
Better to add some comments to explain this here

Reply:
added.

Reply:
If the backslash is followed by another backslash, like `"9\ttab\\\\	tab\n"`, will it return `"9\ttab\\	tab\n"`  incorrectly ?

Reply:
@cfzjywxk it will return `"9	tab\\	tab"` since `\t` also mean a tab